clouddriver-azure/clouddriver-azure.gradle

```diff
@@ -13,8 +13,13 @@ dependencies {
   implementation "org.codehaus.groovy:groovy-all"
   implementation "org.springframework.boot:spring-boot-actuator"
   implementation "org.springframework.boot:spring-boot-starter-web"
-  implementation 'com.microsoft.azure:adal4j:1.6.3'
-  implementation 'com.microsoft.azure:azure:1.35.0'
+  implementation "com.azure.resourcemanager:azure-resourcemanager:2.19.0"
+  implementation "com.azure:azure-identity:1.6.0"
+  implementation "com.azure:azure-storage-blob:12.19.1"
+  implementation("io.projectreactor:reactor-core:3.4.23") {
+    force = true
```

Comment:
is not transitive?

Reply:
hi @nemesisOsorio. I forced Gradle to use this version because I got a class not found error when running the app. 

![image](https://user-images.githubusercontent.com/105648914/194487591-615fa637-db0d-4dd8-9dd6-f47e2b3bfdeb.png)

When I ran the `dependencyInsight` command, the 3.3.17, RELEASE is used in the project, which is used from kork-bom. -> the azure sdk is using 3.4.23 version that's why I forced the module to use that particular version